src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/util/HeapUtil.java

```diff
@@ -0,0 +1,22 @@
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.decisionmaker.util;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class HeapUtil {
+    private static final Logger LOG = LogManager.getLogger(HeapUtil.class);
+
+    public static double getHeapUsedPercentage(final Long heapMaxInBytes, final Long heapUsedInBytes) {
```

Comment:
I am thinking should we make the arguments primitive type and avoid the null check ?

Reply:
yes, just need to avoid the 0/0 case.